src/cookieSyncManager.interfaces.ts

```diff
@@ -1,16 +1,25 @@
-import { MPID } from "@mparticle/web-sdk";
-import { Dictionary } from "./utils";
-import { IConsentRules } from "./consent";
+import { MPID } from '@mparticle/web-sdk';
+import { Dictionary } from './utils';
+import { IConsentRules } from './consent';
 
 export interface ICookieSyncManager {
-    attemptCookieSync: (previousMPID: MPID, mpid: MPID, mpidIsNotInCookies: boolean) => void;
+    attemptCookieSync: (
+        previousMPID: MPID,
+        mpid: MPID,
+        mpidIsNotInCookies: boolean
+    ) => void;
     performCookieSync: (
         url: string,
-        moduleId: number,
+        moduleId: string,
```

Comment:
when i pull a config down that has pixelConfigs in it, [such as this one ](https://jssdkcdns.mparticle.com/JS/v2/beab3f4d34281d45bfcdbbd7eb21c083/config?env=0), it shows that moduleId comes back as a number.  Any reason for this change?  or is it because a test or another internal use of this is wrong?

Reply:
oh i see.  you're turning into a string earlier on.  i'm fine with that